### PR TITLE
NOTIF-521 Adds page to validate the notification message

### PR DESCRIPTION
**Pull request by josejulio**
Adds a page to validate the message - this should help to reduce the confusion when sending messages or wondering why it's invalid.

Screenshot_2022-04-27_11-08-34

Peek 2022-04-27 07-49

Depends on https://github.com/RedHatInsights/insights-schemas-java/pull/40

### CODE REVIEW

**Comment by josejulio** [comment]
@gwenneg could you check the backend changes? 
@aferd could you check the frontend changes?
Thanks!

File: backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/MessageValidationResponse.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public MessageValidationResponse() {
+    }
+
+    public Map<String, List<String>> getErrors() {
+        return errors;
+    }
+
+    public void addError(String field, String message) {
+        this.errors.computeIfAbsent(field, s -> new ArrayList<>());
+        this.errors.get(field).add(message);
```
Comment (gwenneg) [inline backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/MessageValidationResponse.java:22]:
```suggestion
        errors.computeIfAbsent(field, s -> new ArrayList<>()).add(message);
```

File: backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/MessageValidationResponse.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MessageValidationResponse {
+    @NotNull
+    private Map<String, List<String>> errors = new HashMap<>();
+
+    public MessageValidationResponse() {
+    }
```
Comment (gwenneg) [inline backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/MessageValidationResponse.java:14]: Is it really needed?

Comment (josejulio) [reply backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/MessageValidationResponse.java:14]: I guess not anymore.

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**